mint-nft/README.md

```diff
@@ -0,0 +1,25 @@
+# mint NFT
+
+mint NFT on LikeCoin chain from metadata or ISCN ID
+
+## Installation
+
+Run with `node.js = 14`
+
+```bash
+npm install
+```
+
+## Configuration
+
+Change the data in `./list.csv` (reference `./list_example.csv`)
+
+Change the data in `./config/config.js`
+
+Where `MNEMONIC` is your mnemonic phrase , `WAIT_TIME` is wait time after console (before send)
```

Comment:
Seems not the case? And it seems there is no config requiring customization in `config.js`

Reply:
updated